src/pagination/pagination.tsx

```diff
@@ -33,23 +35,46 @@ const Component = (
         pageSize = 10,
         totalItems,
         activePage,
+        pageSizeOptions,
         showFirstAndLastNav,
+        showPageSizeChanger = false,
         onPageChange,
+        onPageSizeChange,
     }: PaginationsProps,
     ref: React.Ref<HTMLDivElement>
 ) => {
     // =============================================================================
     // CONST, STATE, REF
     // =============================================================================
+    const options: DropdownItemProps[] = pageSizeOptions
```

Comment:
you can simply assign `DEFAULT_OPTIONS` as a default value when destructuring on line 38